test/linodes/linodes.bats

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bats
+
+load '../test_helper/bats-support/load'
+load '../test_helper/bats-assert/load'
+load '../common'
+
+@test "it should create linodes with a label" {
+    run linode-cli linodes create --type g6-standard-2 --region us-east --image $test_image --label cli-1 --root_pass $random_pass --text --delimiter "," --no-headers
+    assert_output --regexp ".*,cli-1,us-east,g6-standard-2,$test_image,.*"
```

Comment:
You could use a `--format 'label,region,type,image'` to remove the need for the `.*`s in this regex

Reply:
ah, good idea!